src/librustc_mir/borrow_check/nll/type_check/input_output.rs

```diff
@@ -63,8 +64,25 @@ impl<'a, 'tcx> TypeChecker<'a, 'tcx> {
             }
         };
 
+        // C-variadic fns also have a `VaList` input that's not listed in the signature
+        // (as it's created inside the body itself, not passed in from outside).
+        let maybe_va_list = if universal_regions.c_variadic {
+            let va_list_did = match self.tcx().lang_items().va_list() {
+                Some(did) => did,
+                None => span_bug!(body.span,
+                                  "`va_list` lang item required for variadics"),
```

Comment:
I’m pretty sure there is a way to request for a `lang_item` that does properly report this error at an upper level (look for `fn require_lang_item`).

Reply:
Oh right I should've switched to that (I think I copy-pasted something I found around).